mkt/webapps/serializers.py

```diff
@@ -135,7 +136,7 @@ class Meta:
             'privacy_policy', 'promo_imgs', 'public_stats', 'release_notes',
             'ratings', 'regions', 'resource_uri', 'slug', 'status',
             'support_email', 'support_url', 'supported_locales', 'tags',
-            'upsell', 'upsold', 'user', 'versions'
+            'upsell', 'upsold', 'user', 'tv_featured', 'versions'
         ]
```

Comment:
I think all tv_featured stuff could and should live in `mkt/tv/serializers.py`, unless you see a reason why not ?